7Q.py

```diff
@@ -0,0 +1,10 @@
+from math import sqrt
+def pythagoras(n):
+    for a in range(1,n+1):
+        for b in range(a,n):
+            c_square = a**2 + b**2
+            c = int(sqrt(c_square))
+            if ((c_square - c**2) == 0):
+                print (a, b, c)
+
```

Comment:
Your code and idea of using sqrt is great but there is one question; is the code is working for the our goals. Absolutely not, because we want the numbers below our given number(n) but in your code it isn't like that. when I give the function `n=100`, the value of c is lots of time above the 100.  This has to be fixed.